Flatpak/org.DolphinEmu.dolphin-emu.metainfo.xml

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2016 Jeremy Newton -->
+<component type="desktop-application">
+  <id>org.DolphinEmu.dolphin-emu</id>
+  <name>Dolphin Emulator</name>
+  <developer id="org.dolphin-emu">
+    <name>Dolphin Emulator Project</name>
+  </developer>
+  <summary>GameCube / Wii</summary>
+  <metadata_license>CC-BY-SA-3.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <content_rating type="oars-1.0"/>
+  <!-- Descriptions taken from Dolphin Homepage -->
+  <description><p>Dolphin is an emulator for two recent Nintendo video game consoles: the GameCube and the Wii. It allows PC gamers to enjoy games for these two consoles in full HD (1080p) with several enhancements: compatibility with all PC controllers, turbo speed, networked multiplayer, and even more!</p></description>
+  <screenshots>
+    <screenshot type="default">
+      <caption>Dolphin's main window</caption>
+      <image type="source">https://raw.githubusercontent.com/flathub/org.DolphinEmu.dolphin-emu/master/screenshots/1.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>In-game</caption>
+      <image type="source">https://raw.githubusercontent.com/flathub/org.DolphinEmu.dolphin-emu/master/screenshots/2.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Grapics configuration</caption>
+      <image type="source">https://raw.githubusercontent.com/flathub/org.DolphinEmu.dolphin-emu/master/screenshots/3.png</image>
+    </screenshot>
+    <screenshot>
+      <caption>Controller configuration</caption>
+      <image type="source">https://raw.githubusercontent.com/flathub/org.DolphinEmu.dolphin-emu/master/screenshots/4.png</image>
+    </screenshot>
+  </screenshots>
+  <launchable type="desktop-id">dolphin-emu.desktop</launchable>
+  <provides>
+    <binary>dolphin-emu</binary>
+    <id>dolphin-emu.desktop</id>
+  </provides>
+  <releases>
+    <release version="2407-253" date="2024-08-24"/>
```

Comment:
I think we can par this down to just 2407 and 2409?

Reply:
Flathub only displays the top one anyway, we can reduce it to a single one.

It's only like this because the flathub bot that submits the new versions doesn't delete the older nodes.

I'm a bit more worried about updating it like I was saying here https://github.com/dolphin-emu/dolphin/pull/13040#issuecomment-2316138250 I'm not sure how to best update it.

Should I keep it flatpak-only for now?

Reply:
Oops, I missed that comment.

I agree that it would be nice to automatically populate this field at build time. [I can see that PCSX2 uses some placeholders here that are replaced by a script at build time](https://github.com/PCSX2/pcsx2/blob/master/.github/workflows/scripts/linux/pcsx2-qt.metainfo.xml.in). Perhaps we can use the same system here too?

Reply:
Great, we'll keep it flatpak-only for now.

We can always move it to generic linux builds in the future if necessary.